caikit/runtime/model_management/model_manager.py

```diff
@@ -112,11 +112,27 @@ def __init__(self):
 
         # Keep track of whether lazy loading is enabled
         self._lazy_load_local_models = runtime_cfg.lazy_load_local_models
-        error.value_check(
-            "<RUN44773514E>",
-            not self._lazy_load_local_models or self._local_models_dir,
-            "Must set runtime.local_models_dir with runtime.lazy_load_local_models",
-        )
+
+        if self._lazy_load_local_models:
+            error.value_check(
+                "<RUN44773525E>",
+                runtime_cfg.local_models_dir is not None,
+                (
+                    "runtime.local_models_dir must be set"
+                    " if using runtime.lazy_load_local_models. "
+                    "local_models_dir is unset in config file"
+                ),
+            )
+
+            error.value_check(
+                "<RUN44773514E>",
+                self._local_models_dir != "",
```

Comment:
nit: I liked `not self._local_models_dir` better, python treats empty strings as false